data/category.go

```diff
@@ -140,6 +150,10 @@ func (s *specification) Seed(cfg write.ClientConfig) error {
 	return nil
 }
 
+func (s *specification) Teardown(cfg write.ClientConfig) error {
```

Comment:
Had it just so that specification implemented the `Spec` interface, probably doesn't need to though.